docs/UserGuide.adoc

```diff
@@ -604,6 +606,47 @@ To exit the program, without using the pointer, use: +
 
 Format: `exit`
 
+// tag::calendar[]
+=== Using the calendar
+
+A calendar is useful for marking the important dates. When you have many deadlines and birthdays to take note, the calendar allows you to view all these important dates at a glance. +
+
+==== Marking the dates
+
+With many contacts and tasks to manage, differentiating the birthdays and deadlines on the calendar can be done with the use of colours.
+
+[NOTE]
+The marking of date is done automatically when a contact/task is added, edited or deleted.
+
+Examples:
+
+* A task has deadline on `17-07-2017`. +
+The corresponding date on the calendar is marked *red*. +
+* A contact has birthday on `09-12-2017`. +
+The corresponding date on the calendar is marked *pink*. +
+* `17-07-2017` is a date that is a task's deadline and a contact's birthday. +
+The corresponding date on the calendar is marked *yellow*. +
+
+__Constraints__
+
+The colour of the date on the calendar is *only updated* when next/previous button for month/year is clicked or when a date on the calendar is clicked.
+
+==== Clicking of date on the calendar
+
+When there is many deadlines or birthdays on the same date, instead of switching between `CommandMode` and using the `find` command, you can just click on the date itself.
+
+Examples:
+
+* `17` on the calendar that represents `17-07-2017` is being clicked. +
+Returns task list and contacts list that contains tasks with deadline on `17-07-2017` and contacts with birthday on `17-07-2017` respectively.
```

Comment:
*a list of tasks with deadlines on 17th Jul 2017 and a list of contacts with birthdays on 17th July 2017, respectively